sass/elements/form.sass

```diff
@@ -544,6 +544,8 @@ $help-size: $size-small !default
   font-size: $size-normal
   position: relative
   text-align: left
+  display: flex
```

Comment:
Why the display: flex?

Reply:
I did a few more tests, I noticed that ``display: flex`` was only needed for my special case. I'm sorry for this mess 😥